### PR TITLE
Made palette set-able in inline-editor

### PR DESCRIPTION
**Pull request by bitflower-bdier**

### CODE REVIEW

File: webcomponents/inline-editor/src/components/inline-editor/deckdeckgo-inline-editor.tsx [hunk 1]
```diff
@@ -17,6 +18,8 @@ export class DeckdeckgoInlineEditor {
 
   @Element() el: HTMLElement;
 
+  @Prop({ mutable: true }) palette: DeckdeckgoPalette[];
```
Comment (peterpeterparker) [inline webcomponents/inline-editor/src/components/inline-editor/deckdeckgo-inline-editor.tsx]: I gonna remove the `mutable: true` after merge as it doesn't seems to be needed

**Comment by peterpeterparker** [comment]
cool cool cool, thx your for the PR @bitflower-bdier 👍

**Comment by peterpeterparker** [comment]
The enhancements have been published respectively color v1.0.0-rc.3-2 and inline-editor v1.0.0-rc.3-3

Nota bene:  the new property in the inline-editor was declared but wasn't used! therefore I fixed the implementation in https://github.com/deckgo/deckdeckgo/pull/529

**Comment by bitflower** [comment]
Awesome, thanks! :-)

**Comment by peterpeterparker** [comment]
Cool! Thx for the feedback

**Pull request merged**